spec/rspec/matchers/dsl_spec.rb

```diff
@@ -7,6 +7,11 @@ def ok
     "ok"
   end
 
+  if RUBY_VERSION.to_f >= 2.7
+    require "rspec/matchers/dsl27_syntax"
+    include RSpec::Matchers::DSL27Syntax
```

Comment:
If this is just loading the test code I think it needs to be named differently so its clear.